pyccel/ast/basic.py

```diff
@@ -16,12 +16,12 @@ def fst(self):
         return self._fst
 
 class PyccelAstNode:
-    stage      = None
-    _shape     = None
-    _rank      = None
-    _dtype     = None
-    _precision = None
-    _order     = None
+    stage       = None
+    _shape      = None
+    _rank       = None
+    _dtype      = None
+    _precision  = None
+    _order      = None
```

Comment:
If we want to be consistent with how the module was written so far, shouldn't we add the line
`_precedence = None` here?

Reply:
I had that line in at one point (that's why `_precedence` was still in the copy operator and why this file had been modified), however given that we have to handle objects that might not have a `precedence` anyway (see above, sympy `Symbol` still exists in the syntactic stage) I came to the conclusion that it wasn't necessary to add the `_precedence` property to Nodes that aren't operators

Reply:
I have now checked out this file from the master branch to revert it to the original version without the additional spaces and `_precedence` in the copy operator